.travis.yml

```diff
@@ -20,10 +19,10 @@ addons:
     - libbluetooth3
     - libbluetooth-dev
     - libudev-dev
+    - libusb-1.0
     - cmake
     - html2text
     - clang-format-3.6
-    - binutils
```

Comment:
@AByzhynar basically the task is about to replay the refactoring that has been done in pre_5_0_0_modifications branch and this one might be wrong, will put it back.